### ISSUE TITLE
How to fix failed map image download?

### ISSUE THREAD

**Issue by coredumperror** [imaginary-png/HuntHelper#33]
A few days ago, my HuntHelper's map images vanished (I think it was after I updated the plugin). I tried to download them again, but I got an error message. Now, I can't seem to get the plugin to even attempt to download the images any more, so I'm completely stuck with no images.

What can I do?

**Comment by imaginary-png** [comment]
What is the error message?

You can try downloading the image manually and putting it in the image/maps folder of the plugin. It should be linked in the download popup window.

**Comment by coredumperror** [comment]
I don't recall the error message, as it was several days ago and I haven't been able to re-trigger it. Checking and un-checking "Map Image" causes nothing to happen except a slight change in opacity of the nap of the location you're in, in the upper left corner of the HH window.

> putting it in the image/maps folder of the plugin

After doing some digging, I finally figured out where this folder is, and I found this:
![image](https://github.com/imaginary-png/HuntHelper/assets/413571/2a7a96d1-508c-4606-a836-dbeaa69c0bf8)
So it looks like I have the map images... They just don't work for some reason.

**Comment by imaginary-png** [comment]
Show me an in-game screenshot of the whole hunthelper map + settings window. 

Is your map opacity 0?

**Comment by any-wrong** [comment]
I met the same problem.It's my screenshot of the whole hunthelper map + settings window.
When I was updated the version 1.0.6 to 1.0.7 and the game patch was 6.4.5.
![hunhelper](https://github.com/imaginary-png/HuntHelper/assets/96103818/f91237c1-62c4-421e-908d-55e0bcd401af)
I tried few times to reinstall the hunthelper and redownload map images but not working.

![hunhelper(2)](https://github.com/imaginary-png/HuntHelper/assets/96103818/8c5515eb-ac35-44a2-acfd-e2406974f00a)
But this time the game patch update to 6.4.8 and retried these operations it is worked!!!

**Comment by imaginary-png** [comment]
Please use /xllog to open the dalamud console  

In /xlplugins, turn off and on HuntHelper  

Then /hh to try and load the map image

Take a screenshot of the console if there are any error messages

**Comment by coredumperror** [comment]
Sorry, but while the second person who chimed in managed to get past it, **I'm** still having this issue. I followed your instructions, and took this screenshot:
![HuntHelper](https://github.com/imaginary-png/HuntHelper/assets/413571/af71e713-b887-4efc-b6fc-aa809ff33104)

There's nothing in the logs decrying any issues with the map images.

**Comment by coredumperror** [comment]
I did finally fix this by not only disabling and uninstalling HuntHelper, but also deleting its entire folder from the plugins directory. After I re-installed, it finally asked again if I wanted to download the map images, and I did so. That finally made the images show up on the map window.

Very strange that the images were failing to load despite being in the right folder, before. No clue what could have caused that. 

Maybe adding a "force re-download of images" feature would be a good idea?
